apps/dashboard/app/mailers/support_ticket_mailer.rb

```diff
@@ -0,0 +1,39 @@
+# Emailer class responsible to send support ticket emails
+# It supports emailer settings in standard Rails configuration
+# or provided in the support_ticket configuration object.
+class SupportTicketMailer < ActionMailer::Base
+  helper :support_ticket
+  default template_path: ["support_ticket/email"]
+
+  def support_email(context)
+    @context = context
+
+    @context.support_ticket.attachments.to_a.each do |request_file|
+      attachments[request_file.original_filename] = File.read(request_file.tempfile)
+    end
+
+    email_service_config = ::Configuration.support_ticket_config.fetch(:email, {})
+
+    # Override Rails defaults with support ticket configuration only if provided
+    # This will allow admins to use standard Rails configuration settings if required.
+    ActionMailer::Base.raise_delivery_errors = email_service_config[:raise_delivery_errors] if email_service_config.key?(:raise_delivery_errors)
+    ActionMailer::Base.perform_deliveries = email_service_config[:perform_deliveries] if email_service_config.key?(:perform_deliveries)
```

Comment:
I don't think we want either of these. I mean, there's no real point in exposing them to the users I don't believe.  It seems like we'd always want to raise errors and perform deliveries.

Reply:
Will remove